middlewares/games.js

```diff
@@ -81,7 +92,11 @@ const checkEmptyFields = async (req, res, next) => {
 };
 
 // Check categories by id
-const checkIfCategoriesAvaliable = async (req, res, next) => {
+const checkIfCategoriesAvailable = async (req, res, next) => {
+  if (req.isVoteRequest) {
+    next();
+    return;
+  }
   if (!req.body.categories || req.body.categories.length === 0) {
     res.setHeader("Content-Type", "application/json");
     res
```

Comment:
```suggestion
    res.status(400)
```

statusCode - такой функции нет

Reply:
Исправил 👌